demos/virtual_ai_assistant_demo/main.py

```diff
@@ -52,27 +53,47 @@ def load_chat_model(model_name: str, token: str = None) -> OpenVINOLLM:
     if not model_path.exists():
         log.info(f"Downloading {model_name}... It may take up to 1h depending on your Internet connection and model size.")
 
+        # Prepare base kwargs
+        kwargs = {
+            "export": True,
+            "compile": False,
+            "load_in_8bit": False,
+            "use_safetensors": True  # Add this to use safetensors format
+        }        
+        
+        if token:
+            kwargs["use_auth_token"] = token
+            os.environ["HUGGING_FACE_HUB_TOKEN"] = token
+
         # openvino models are used as is
         is_openvino_model = model_name.split("/")[0] == "OpenVINO"
         if is_openvino_model:
-            chat_model = OVModelForCausalLM.from_pretrained(model_name, export=False, compile=False, token=token)
-            chat_model.save_pretrained(model_path)
-        else:
-            chat_model = OVModelForCausalLM.from_pretrained(model_name, export=True, compile=False, load_in_8bit=False, token=token)
+            kwargs["export"] = False
+        
+        chat_model = OVModelForCausalLM.from_pretrained(model_name, **kwargs)
+        chat_model.save_pretrained(model_path)
```

Comment:
The model here doesn't have to be saved if the quantization will follow. Please restore the previous approach.